packages/schemas/src/types/logto-config/jwt-customizer.ts

```diff
@@ -19,7 +26,25 @@ export enum LogtoJwtTokenKeyType {
   ClientCredentials = 'client-credentials',
 }
 
+export type JwtCustomizerUserContext = UserInfo & {
```

Comment:
If not ts IntelliSense is provided, no need to define the type exclusively?

Reply:
When developing, is it more convenient to see type hints in places where the type is referenced?

Reply:
sound reasonable.